pyproject.toml

```diff
@@ -303,16 +304,14 @@ ignore = [
   "PLW0603",
   # Docstrings should not be included in stubs
   "PYI021",
+  # Use `typing.NamedTuple` instead of `collections.namedtuple`
+  "PYI024",
```

Comment:
I assume that there might be objections against enabling this rule as it might(?) affect runtime a tiny bit. Could move it to the todo list if there are no objections (currently in the ignore list).

Reply:
Would the runtime be present in the importing of NamedTuple? I would have thought that would be protected in `if TYPE_CHECKING`?

Reply:
I believe it would have runtime implications (but they are likely to be neglectable) as you have to define a class that is used at runtime https://docs.python.org/3/library/typing.html#typing.NamedTuple